src/screens/wallet/token-list/token-list-item/token-list-item.styles.ts

```diff
@@ -1,9 +1,11 @@
 import { createUseStyles } from '../../../../styles/create-use-styles';
+import { formatSize } from '../../../../styles/format-size';
 
 export const useTokenListItemStyles = createUseStyles(({ colors, typography }) => ({
   rightContainer: {
     justifyContent: 'center',
-    alignItems: 'flex-end'
+    alignItems: 'flex-end',
+    maxWidth: formatSize(150)
```

Comment:
your solution will truncate token name on the laptop.  
fix this using `flex` properties.